src/crypto.js

```diff
@@ -14,15 +16,16 @@ export function stretchPassphrase (passphrase: string | Buffer, { salt, n, r, p
   return { key, salt }
 }
 
-export function aesEncrypt (key: Buffer, message: Buffer, opts = { cipher: 'aes-256-gcm', iv: crypto.randomBytes(IV_LEN_BYTES) }) : Object {
-  const cipher = crypto.createCipheriv(opts.cipher, key, opts.iv)
+export function aesEncrypt (key: Buffer, message: Buffer) : Object {
+  const iv = randomBytes(IV_LEN_BYTES)
+  const cipher = aes.createCipheriv('aes-256-gcm', key, iv)
   const blob = Buffer.concat([cipher.update(message), cipher.final()])
   const authTag = cipher.getAuthTag()
-  return { authTag, blob, iv: opts.iv }
+  return { authTag, blob, iv }
 }
 
-export function aesDecrypt (key: Buffer, blob: Buffer, { cipher = 'aes-256-gcm', iv, authTag } = {}) : Buffer {
-  const decipher = crypto.createDecipheriv(cipher, key, iv)
+export function aesDecrypt (key: Buffer, blob: Buffer, { iv, authTag } = {}) : Buffer {
```

Comment:
can iv or authTag be undefined?
```suggestion
export function aesDecrypt (key: Buffer, blob: Buffer, { iv, authTag }) : Buffer {
```

Reply:
No, good idea.